src/ergw_aaa_radius.erl

```diff
@@ -255,6 +296,45 @@ gethostbyname(Name) ->
 	    Other
     end.
 
+avp_filters(#{avp_filter := Filters}, Attrs) ->
+    lists:filter(
+      fun({{Vendor, _} = Id, _}) ->
+	      (not maps:is_key(Id, Filters)) orelse
+	      (not maps:is_key({vendor, Vendor}, Filters));
+	 ({Id, _}) ->
+	      not maps:is_key(Id, Filters)
+      end, Attrs).
+
+vendor_dicts(#{vendor_dicts := Vendors}, Session, Attrs) ->
+    lists:foldl(fun(Vendor, A) -> vendor_dict(Vendor, Session, A) end, Attrs, Vendors).
+
+vendor_dict(?'Ituma', Session, Attrs) ->
+    IEs = ['UE-Wwan-RSSI', 'LI-Location', 'Wifi-Connect-Typ', 'SSID',
+	   'Called-Station-Id'],
+    lists:foldl(fun(IE, A) -> vendor_ituma(IE, Session, A) end, Attrs, IEs);
+vendor_dict(_, _, Attrs) ->
+    Attrs.
+
+vendor_ituma('UE-Wwan-RSSI', _Session, Attrs) ->
```

Comment:
Do we need this clause? As it does nothing and just returning `Attrs` as the last one `vendor_ituma` clause just does the same. Or there will be some things here in future?

Reply:
It in the feature requests, but deemed to be postponed as it would need a firmware change. So this is placeholder for things that need to added in the future.
I've put it there mainly as a reminder that some desired things are still missing and can not be implemented at this time.

On the other hand, ergw_aaa could have a implementation for this IE, even when the main capwap code does not supply this value.

I'll add something for this here.